submissions/OgeEjim/DOM/productpricecalculation.js

```diff
@@ -0,0 +1,72 @@
+document.addEventListener("load", initialise());
+
+var initialCost;
+var initialQty;
+var quantity;
+var total;
+var shippingCost;
+var price;
+var buyButton;
+var errorMsg;
```

Comment:
Instead of declaring these variables here and then using another function to declare the value of the variables, why not just declare the values of the variables up here once and forall and then use it in your code. This will make your code shorter. The shorter, the better.

Reply:
Ma I get this but It took a long time to get this one to work, my shorter attempts were not working properly. I'll still look at it, but I wouldn't like it to hold me back much longer.